vars/tdr.groovy

```diff
@@ -33,6 +33,16 @@ def reportFailedBuildToGitHub(String repo) {
   }
 }
 
+def assembleAndStash(String libraryName) {
+  sh "sbt -no-colors assembly"
+  stash includes: "target/scala-2.13/${libraryName}.jar", name: "${libraryName}-jar"
```

Comment:
Is it worth adding "scala-2.13" as a "constant" to the library as its used in multiple places. Also would make it easier if the scala version is ever updated.

Reply:
Makes sense, I've added a field at the top for the scala version. I've tried it by replaying an existing job with this in and it seems to work.